docker/app/000-default.conf

```diff
@@ -0,0 +1,35 @@
+ServerName 127.0.0.1
```

Comment:
I'm not okay to use apache, I would like to keep nginx (and this is something important to me :) ).